modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -304,8 +299,8 @@ const SelectMenu = (props: SelectMenuProps) => {
   // Render nothing if buttonRef.current hasn't been set
   return buttonRef.current ? (
     <Popper
-      placement={isFlipped ? 'top-start' : 'bottom-start'}
-      open={!isHidden}
+      placement={isFlipped ? 'top' : 'bottom'}
+      open={!isHidden && width !== 0}
```

Comment:
This makes sure the Popper is not activated until the width of the button has been calculated since we're using `top` and `bottom` which are "centered" Popper types.

`top-start` and `bottom-start` are incorrectly rounded which causes problems for HighDPI displays in Firefox and Edge. The problem comes from the following lines in Popper.js: https://github.com/popperjs/popper-core/blob/master/src/utils/computeOffsets.js#L68-L81

Reply:
Removed the `width !== 0` check in #743 since it's no longer necessary now that we're using `useLayoutEffect` instead of `useEffect` when setting the width of the menu (the check was preventing a newly activated menu from scrolling to the selected item if that item was initially out of view).